backend/postgresql/schema.sql

```diff
@@ -62,7 +31,19 @@ CREATE TYPE valideducationlevel AS ENUM(
     'OTHER'
 );
 CREATE CAST (character varying AS valideducationlevel) WITH INOUT AS ASSIGNMENT;
-CREATE TABLE answer_educationlevel(
-    userid UUID REFERENCES users(id) UNIQUE,
-    response valideducationlevel
+
+CREATE TABLE answers(
+    userid UUID REFERENCES users(id) UNIQUE,
+    answer_jobacceptdate TIMESTAMPTZ,
+    answer_jobsearchstartdate TIMESTAMPTZ,
+    answer_workmodel validworkmodel,
+    answer_workcontract validworkcontract,
+    answer_jobapplicationcount INT CHECK (
+        answer_jobapplicationcount BETWEEN 0 and 10000
+    ),
+    answer_jobtitle VARCHAR(50),
+    answer_yearsofproexperience INT CHECK (
+        answer_yearsofproexperience BETWEEN 0 and 50
+    ),
+    answer_educationlevel valideducationlevel
```

Comment:
very nice, who's the previous idiot who programmed individual tables for each answer? (i'm the previous idiot)